apps/resources/cs.json

```diff
@@ -12,10 +12,20 @@
     "x86_64-pc-linux": "gz+https://github.com/coursier/coursier/releases/download/v${version}/cs-x86_64-pc-linux.gz",
     "aarch64-pc-linux": "gz+https://github.com/VirtusLab/coursier-m1/releases/download/v${version}/cs-aarch64-pc-linux.gz",
     "x86_64-apple-darwin": "gz+https://github.com/coursier/coursier/releases/download/v${version}/cs-x86_64-apple-darwin.gz",
-    "aarch64-apple-darwin": "gz+https://github.com/VirtusLab/coursier-m1/releases/download/v${version}/cs-aarch64-apple-darwin.gz",
+    "aarch64-apple-darwin": "gz+https://github.com/coursier/coursier/releases/download/v${version}/cs-aarch64-apple-darwin.gz",
```

Comment:
We still have the linux ones, but I guess that would be more difficult to replace?

Reply:
I used to build those on the coursier CI, with an action using qemu and docker under-the-hood on x86-64 workers, but it was pretty slow.

[Apparently](https://github.com/orgs/community/discussions/19197), Linux ARM workers should arrive at the end of the year for open source projects

Reply:
Is it a big problem for you to keep this one?

Reply:
And it seems one can't run docker on Mac ARM instances of GitHub actions (according to [this](https://github.com/douglascamata/setup-docker-macos-action?tab=readme-ov-file#arm64-processors-m1-m2-m3-series-used-on-macos-14-images-are-unsupported)), like is done to build the LInux ARM image in VirtusLab/coursier-m1

Reply:
> Is it a big problem for you to keep this one?

It's fine, just wanted to reduce the maintenance burden, but if it's not possible currently than we can wait for the linux arm runners.